src/components/Modal.tsx

```diff
@@ -4,23 +4,23 @@ import { MenuIcon } from "~/icons/Menu";
 import ReactMarkdown from "react-markdown";
```

Comment:
Can we rename this file & component to Comment Modal? 

The reason is; there are other modals in the codebase (like the modal from react-modal) used and this naming can help distinguish

Reply:
done!